packages/concerto-core/test/models/model.js

```diff
@@ -83,75 +83,75 @@ describe('Model Tests', function(){
             // o String stringProperty
             resource.setPropertyValue('stringProperty', 'string');
             resource.stringProperty.should.equal('string');
-            assert.throws( function() {resource.setPropertyValue('stringProperty', 1);}, /ValidationException: Model violation in the "org.acme.base.BaseAsset#123" instance. The field "stringProperty" has a value of "1" \(type of value: "number"\). Expected type of value: "String"./);
+            assert.throws( function() {resource.setPropertyValue('stringProperty', 1);}, /ValidationException: Model violation in the "org.acme.base.BaseAsset#string" instance. The field "stringProperty" has a value of "1" \(type of value: "number"\). Expected type of value: "String"./);
```

Comment:
I don't really understand why the id in these tests changed from the "123" that was passed in when creating the instance to instead the string "string". Is this expected?

Reply:
It's because (rather confusingly) the definition of BaseAsset is
```
asset BaseAsset identified by stringProperty {
  o String stringProperty
  //...
``` 

So now, changing the value of that property, also has the effect of changing the identifier for the asset. This should have previously been considered a bug IMO!